components/shell/CommandPalette/useAsyncSearch.tsx

```diff
@@ -299,7 +299,17 @@ export const useCommandPaletteAsyncSearch = () => {
             // When we have an exact content by rkey, we want to prioritize it over action for the account by did
             priority: 6,
             perform: () => {
-              router.push(`?quickOpen=${atUri}`)
+              if (fragments.did) {
+                return router.push(`?quickOpen=${atUri}`)
+              }
+
+              const { handle } = fragments
+              if (handle) {
+                getDidFromHandle(handle).then((did) => {
+                  if (!did) return
+                  router.push(`?quickOpen=${atUri.replace(handle, did)}`)
+                })
+              }
```

Comment:
Is this related to the OAuth integration, does it fix a bug, or a misc change/improvement?

Reply:
@foysalit , I believe this was you ?

Reply:
yep, this is a minor bugfix that I noticed while testing this branch so just put it in here. 
essentially, at-uris can contain handle instead of DID so this just makes sure that we resolve the handle to DID before doing the reroute since ozone always wants to use DIDs in URL.